examples/plot_clip.py

```diff
@@ -0,0 +1,125 @@
+import earthpy.clip as ec
+
+###############################################################################
+# Import Example Data
+# -------------------
+#
+# Once the packages have been imported, download the data needed for this
+# example: one line shapefile containing all major roads in North America,
+# and one polygon shapefile containing the United States boundary.
+
+data = et.data.get_data("spatial-vector-lidar")
+
+###############################################################################
+# Open Files with GeoPandas and Reproject the Data
+# -------------------------------------------------
+#
+# Start by setting your working directory. Then, import the data files to
+# GeoDataFrames using GeoPandas.
+#
+# Recall that the data must be in the same CRS in order to use the
+# ``clip_shp()`` function. If the data are not in the same CRS, be sure to use
+# the ``to_crs()`` function from GeoPandas to match the projects between the
+# two objects, as shown below.
+
+# Set your home environment
+os.chdir(os.path.join(et.io.HOME, "earth-analytics"))
+
+# Open both files with GeoPandas
+roads = gpd.read_file(
+    os.path.join(
+        "data",
+        "spatial-vector-lidar",
+        "global",
+        "ne_10m_roads",
+        "ne_10m_n_america_roads.shp",
+    )
+)
+country_boundary = gpd.read_file(
+    os.path.join(
+        "data", "spatial-vector-lidar", "usa", "usa-boundary-dissolved.shp"
+    )
+)
+
+# Reproject the roads layer to match the US boundary CRS
+roads = roads.to_crs(country_boundary.crs)
+
+# Plot the unclipped data
+# The plot below shows the roads data before it has been clipped
+fig, ax = plt.subplots(figsize=(12, 8))
+roads.plot(ax=ax)
+
+# Use the .boundary option in GeoPandas plotting to just plot the boundary of a polygon, instead of
+# a filled in version of the polygon. This allows for other data, such as the roads data, to be seen
+# in reference to the polygon boundary.
+country_boundary.boundary.plot(ax=ax, color="red")
+ax.set_title("Major NA Roads Unclipped to US Border", fontsize=20)
+ax.set_axis_off()
+plt.show()
+
+###############################################################################
+# Clip the Data
+# --------------
+#
+# Now that the data are opened as GeoDataFrame objects and in the same
+# projection, the data can be clipped! Recall that in this example, the roads
+# will be clipped to the United States boundary. To clip the data, make
+# sure you put the object to be clipped as the first argument in
+# ``clip_shp()``, followed by the vector object (boundary) to which you want
+# the first object clipped. The function will return the clipped GeoDataFrame
+# of the object that is being clipped (e.g. roads).
+
+roads_clipped = ec.clip_shp(roads, country_boundary)
+
+# Plot the clipped data
+# The plot below shows the results of the clip function applied to the roads
+# sphinx_gallery_thumbnail_number = 5
+fig, ax = plt.subplots(figsize=(12, 8))
+roads_clipped.plot(ax=ax)
```

Comment:
can't you add some parameters to .plot() ? i'm pretty sure you can.

http://geopandas.org/gallery/plotting_with_geoplot.html

let's try to simplify this code a bit if it's just a quick plot that needs a title and such.